rules/falco_rules.yaml

```diff
@@ -2676,7 +2676,7 @@
   tags: [file, mitre_persistence]
 
 - list: remote_file_copy_binaries
-  items: [rsync, scp, sftp, dcp]
+  items: [rsync, scp, sftp, dcp, wget, scp]
```

Comment:
`scp` is duplicated

Reply:
@Kaizhe thanks for the catch. Corrected by replacing `scp` with `curl`